index.js

```diff
@@ -224,6 +243,17 @@ class EthicalAdsWarning extends Error {}
  */
 export var wait;
 
+/* Loading placements manually rather than the normal way
+ *
+ *   <div data-ea-publisher="..." data-ea-manual="true"></div>
+ *   <script>
+ *     ethicalads.load();
```

Comment:
Does this allow us to set keywords on the request? That seems important since we don't have that data prior?

Reply:
We could probably make it accept options but it doesn't currently. You can still attach the keywords to the div the normal way.

Reply:
Right, but I thought the point of this PR was to use it on RTD, where we don't know the keywords until we get the API response?

Reply:
My plan was to add/modify that DOM element after the API response.

Reply:
I'll test this out and see how well it works. Maybe we do need additional options to `load()`.